File: algoritmo_e_logica/python/q_8.py
def subtracao(a, b):
    return a - b

# Lista de tuplas com pares de números
exemplos = [
    (15, 3),
    (28, 7)
]

# Iterando pela lista e aplicando a função
for a, b in exemplos:
    print(f'A subtração {a} - {b} = {subtracao(a, b)}')

File: algoritmo_e_logica/javascript/q_8.js
// Crie uma função chamada Subtracao que receba dois números como parâmetros e retorne o resultado da subtração entre eles

function subtracao(a, b) {
    return a - b;
}

let exemplos = [
    [10, 3],
    [25, 7]
];

exemplos.forEach(par => {
    console.log(`A subtração ${par[0]} - ${par[1]} = ${subtracao(par[0], par[1])}`);
});